### DISCUSSION TITLE
How can I force to clean nuget library for .Net Interactive?

### DISCUSSION THREAD

**Discussion by young-kcd** [dotnet/interactive#2047] [Q&A] [answered]
Is there any way to clean nuget library?

I'm facing weird error.

works fine on one of my PC but not work on some with same code.
```
#r "nuget:Newtonsoft.Json"

using Newtonsoft.Json;

public class A
{
    public string a { get; set; }
}

var a = JsonConvert.SerializeObject(new A());
display(a);
var a_d = JsonConvert.DeseiralizeObject<A>(a);
display(a_d);
```

the error is

`(10,23): error CS0117: 'JsonConvert' does not contain a definition for 'DeseiralizeObject'`

I restarted the PC which error occurs, but not fixed

Is there any way to force to clean nuget library?

** environment
cmd> dotnet tool update -g Microsoft.dotnet-interactive
Tool 'microsoft.dotnet-interactive' was reinstalled with the latest stable version (version '1.0.325601').
cmd> dotnet --version
6.0.300
Windows 10 64bit

**Comment by jonsequitur** [comment, marked as answer]
You've misspelled `DeserializeObject`.

**Reply by young-kcd** [reply]
I'm an idiot..... Sorry and thanks!!!!!!!!!

**Reply by jonsequitur** [reply]
No problem at all. Usually people complain when I correct their spelling. 😂
